projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/components/update-flag/update-flag.component.spec.ts

```diff
@@ -134,4 +137,24 @@ describe('UpdateFlagComponent', () => {
 
     expect(displayedStatuses).toEqual(Object.values(CaseFlagStatus));
   });
+
+  it('should start with currentFormStep as FLAG_FORM', () => {
+    expect(component.currentFormStep === UpdateFlagStatesEnum.FLAG_FORM);
+  });
+
+  it('should set currentFormStep to WELSH_TRANSLATION_FORM when clicking onNext() if welsh checkbox is selected', () => {
+    component.formGroup.get(component.FLAG_WELSH_TRANSLATION_CONTROL_NAME).setValue(true);
+    fixture.detectChanges();
+    component.onNext();
+
+    expect(component.currentFormStep === UpdateFlagStatesEnum.WELSH_TRANSLATION_FORM).toBe(true);
+  });
+
+  it('should set currentForm back to FLAG_FORM if it`s WELSH_TRANSLATION_FORM', () => {
```

Comment:
@MateiHMCTS - would favour using a backslash to escape the single quote mark for `it's` (i.e. `it\'s`) rather than using the backtick